src/content/blog/rust-based-javascript-linters-lack-typed-linting-right-now/index.mdx

```diff
@@ -0,0 +1,151 @@
+---
+description: "Explaining why the speed gains from Rust linters aren't comparable to the full feature set of typescript-eslint."
+image:
+  alt: Josh smiling on stage and giving a presentation showing the "Type Checking React (v2)" from the linked slides.
+  src: react-miami-me-speaking.jpg
+pubDate: 2024-01-07
+tags: [
+	"biome",
+	"deno lint",
+	"eslint",
+	"ezno",
+	"linting",
+	"oxc",
+	"rslint",
+	"stc",
+	"typed linting",
+	"typescript-eslint",
+]
+title: "Rust-Based JavaScript Linters Lack Typed Linting Right Now"
+---
+
+One of 2023's biggest trends for web tooling was rewriting existing tooling in [Rust](https://www.rust-lang.org).
+Rust is a wonderful programming language that allows for shockingly fast binaries that still interop well with other web tools courtesy of WebAssembly.
+The speedups seen in tools such as [swc](https://swc.rs) and [Turbopack](https://turbo.build/pack) are very exciting for fast development experiences.
+
+[Biome](https://biomejs.dev), [`deno lint`](https://lint.deno.land), [oxc](https://github.com/oxc-project/oxc), and [RSLint](https://rslint.org) are all projects that include at least a JavaScript/TypeScript linter written in Rust.
```

Comment:
Good note from Victorien to mention: Biome has >=97% compatibility with Prettier. https://prettier.io/blog/2023/11/27/20k-bounty-was-claimed